packages/DatePicker/src/components/Calendar/Calendar.js

```diff
@@ -10,17 +10,28 @@ import ArrowLeft from "@paprika/icon/lib/ArrowLeft";
 import ArrowRight from "@paprika/icon/lib/ArrowRight";
 import useI18n from "@paprika/l10n/lib/useI18n";
 
-import CalendarStyled, { DayTriggerStyle, CalendarHeaderStyled } from "./Calendar.styles";
+import ShortcutPanel from "../ShortcutPanel";
+import { isMomentObjectOrNull } from "../../helpers";
+import CalendarStyled, {
+  CalendarWrapperStyled,
+  DayTriggerStyle,
+  MonthHeaderButtonStyled,
+  ArrowIconStyles,
```

Comment:
I find the mix of `Styles` and `Styled` a bit odd... are you planning to refactor all to `Styles`?

Reply:
Yes, I already finish it in my local branch, will push the changes today